doc/roadmap.md

```diff
@@ -58,6 +59,29 @@ of the product even though development of these features has not yet been
 scheduled for a release in the near future. Please feel free to contact us if
 you wish to get involved in the implementation or influence the roadmap
 
-- Dynamic / high order attribute values (e.g. an attribute being a sum of two other attributes) ([#3815](https://github.com/telefonicaid/fiware-orion/issues/3815))
 - Performance improvements in the notifications system ([#3461](https://github.com/telefonicaid/fiware-orion/issues/3461))
-- Improve performance in update/notifications (connection-oriented, lightweight ingestion, etc.)
+- Improve performance in update (connection-oriented, lightweight ingestion, etc.)
+
+## Features included
+
+The following list contains all features that were in the roadmap and have already been implemented.
+
+- Per sub/reg HTTP timeout ([#3842](https://github.com/telefonicaid/fiware-orion/issues/3842))
```

Comment:
It could be a good idea to include the version in which the feature was completed. For instance:

```suggestion
- Per sub/reg HTTP timeout ([#3842](https://github.com/telefonicaid/fiware-orion/issues/3842)) (X.Y.Z)
```

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4077/commits/c8bb35739ec2810f311add702492ec774c8b9c7f